apps/data/src/pipelines/centrum_pipeline.py

```
@@ -782,6 +784,113 @@ def execute_macro_udf(pdf: pd.DataFrame) -> pd.DataFrame:
+        "delta.enableChangeDataFeed": "true",
+        "delta.feature.variantType-preview": "supported",
+    },
+)
+def experiment_macro_data_sandbox():
+    """Process macros via backend API (Lambda sandbox) for comparison."""
+
+    base_df = (
+        dlt.read_stream(EXPERIMENT_RAW_DATA_TABLE)
+        .filter("macros IS NOT NULL")
+        .filter("size(macros) > 0")
+        .filter(~F.coalesce(F.col("skip_macro_processing"), F.lit(False)))
```

Ruff (0.15.10)

[warning] Boolean positional value in function call

(FBT003)